app/src/androidTest/java/com/duckduckgo/app/tabs/model/TabDataRepositoryTest.kt

```diff
@@ -229,6 +229,27 @@ class TabDataRepositoryTest {
         assertTrue(captor.firstValue.position == 1)
     }
 
+    @Test
```

Comment:
I have changed this a bit, it now uses `blockingObserve` to observe the tab selected live data in order to do the testing. The setup is pretty awful but the test much easier, I don't really want to spend time right now changing the other tests but I think it would be a good idea to move away from mocking the dao as well. If you like this I'll create a LHF task to change the other tests and remove the duplicated code that creates the database, etc.